src/idl_gen_swift.cpp

```diff
@@ -1869,6 +1831,17 @@ class SwiftGenerator : public BaseGenerator {
     }
   }
 
+  std::string SwiftConstant(const FieldDef& field) {
+    const auto default_value =
+        StringIsFlatbufferNan(field.value.constant) ? ".nan" :
```

Comment:
do we have `.nan` in swift?

Reply:
Yes, we do: https://developer.apple.com/documentation/swift/floatingpoint/nan.

Reply:
Oh! that's cool!